rest/encoded_response_writer.go

```diff
@@ -121,6 +151,11 @@ func (w *EncodedResponseWriter) Close() {
 	}
 }
 
+// isHijackable determines if the underlying writer implements hijack interface
+func (w *EncodedResponseWriter) isHijackable() bool {
+	return false
```

Comment:
What makes this non-hijackable? Isn't there a potentially hijackable ResponseWriter inside the EncodedResponseWriter?